src/main/java/com/google/sps/utils/UserUtils.java

```diff
@@ -47,25 +75,24 @@
     * @param  entityPropertyValue  the value of the property that is filtered for
     * @return                      a single entity that has the same value for the property in the parameter 
     */
+    
   public static Entity getEntityFromDatastore(String entityName, String entityPropertyTitle, String entityPropertyValue, DatastoreService datastore) {
 
     if(entityPropertyValue == "" || entityName == "" || entityPropertyTitle == "" || datastore == null){
         return null;
     }
-
+ 
```

Comment:
Your editor probably has a "remove trailing whitespace" button somewhere. Please find and depress that button.